src/gossip/service.zig

```diff
@@ -132,13 +133,27 @@ pub const GossipService = struct {
     const Entrypoint = struct { addr: SocketAddr, info: ?ContactInfo = null };
 
     pub fn init(
-        allocator: std.mem.Allocator,
-        my_contact_info: ContactInfo,
-        my_keypair: KeyPair,
-        entrypoints: ?[]const SocketAddr,
-        exit: *AtomicBool,
-        logger: Logger,
+        params: struct {
```

Comment:
can we keep the params as params for now - not sure how we feel about making them structs for rn - likely something we should discuss more

Reply:
Sure, that's fine. I did it here because having two unnamed allocator parameters would be confusing, but that's less important to me than getting this merged.